### ISSUE TITLE
question about the results of VL-SA T model

### ISSUE THREAD

**Issue by qaz88666** [wz7in/CVPR2023-VLSAT#13]
Hi, Thanks for releasing the code! I have a question about the results of the VL-SAT model. During training, every 10 epochs of the network, if the relevant metrics tested on the test set are better than the results of the last test set, it is updated to the best model, thus saving the best model iteratively. Does this treat a test set as a validation set? Is that reasonable? I ran the VL-SAT and got the best model results, but when I tested it again with the best model, the results were slightly different from the test set I got in 10 training sessions. May I ask why?
